### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by sunasing**

### CODE REVIEW

**Comment by sunasing** [comment]
#sign-off

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ By default, Defender for Endpoint for iOS includes and enables the web protectio
 
 While enabled by default, there might be some cases that require you to disable VPN. For example, you want to run some apps that do not work when a VPN is configured. In such cases, you can choose to disable VPN from the app on the device by following the steps below:
 
-1. On your iOS device, open the **Settings** app and click or tap  **VPN**.
+1. On your iOS device, open the **Settings** app and click or tap **General** and then **VPN**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:71]: Just to make it slightly more natural to read:
```suggestion
1. On your iOS device, open the **Settings** app, click or tap **General**, and then **VPN**.
```

**Review by illfated — APPROVED** [review]
Looks good to me otherwise.

**Comment by sunasing** [comment]
#sign-off

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**